lib/client/.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": [
```

Comment:
When not specifying `root: true`, the extends is not necessary as eslint reads up the folders until it finds a root eslintrc.
See: https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy